content/altinn-studio/v9/getting-started/create-user/_index.nb.md

```
@@ -6,43 +6,46 @@ description: Slik lager du deg en bruker i Altinn Studio
+header: 'Lag en bruker'
+footer: 'Altinn Studio - kom i gang'
 ---
-
-## Lag en bruker i Altinn Studio
-
+Alle som ønsker å teste og/eller bruke Altinn Studio må først lage en bruker. Dette er selvbetjent.
 Din Altinn Studio-bruker er personlig for deg og kan knyttes til en eller flere organisasjoner for å samarbeide med andre og få tilgang til eksisterende apper.
+Om du skal ha tilgang til å lage tjenester for en organisasjon må du kontakte den som administrerer Altinn Studio
+for din organisasjon.
 
 ### Lag en bruker med Ansattporten
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)